ccmlib/scylla_node.py

```diff
@@ -191,6 +191,13 @@ def _start_scylla(self, args, marks, update_pid, wait_other_notice,
         scylla_log = open(log_file, 'a')
         env_copy = os.environ
         env_copy['SCYLLA_HOME'] = self.get_path()
+        dbuild_so_dir = os.environ.get('SCYLLA_DBUILD_SO_DIR')
+        if dbuild_so_dir:
+            executable = os.path.join(dbuild_so_dir, 'scylla.sh')
+            with open(executable, 'w+') as f:
+                f.write('#!/bin/bash\nexec -a scylla ' + ' '.join([os.path.join(dbuild_so_dir, 'ld-linux-x86-64.so.2'), '--library-path', dbuild_so_dir]) + ' "$@" ')
```

Comment:
This could be racy with multiple nosetests processes running in parallel.
How about preparing this script externally and just running it here?

Reply:
it would make it a change both here and dtest. maybe just check if file exists before creating should be enough ?

Reply:
> it would make it a change both here and dtest. maybe just check if file exists before creating should be enough ?

ok, and add a FIXME comment that this is a temporary measure for  #149 until we add support for scylla relocatable package.

Reply:
Done